video-course/src/app/core/header/logoff/logoff.component.ts

```diff
@@ -7,12 +8,12 @@ import { Component, OnInit } from '@angular/core';
 })
 export class LogoffComponent implements OnInit {
 
-  constructor() { }
+  constructor(private authService: AuthenticationService) { }
 
   ngOnInit(): void {
   }
 
   onLogoffClick() {
-    console.log('Logoff was clicked');
+    this.authService.logOff();
   }
 }
```

Comment:
this may be too much to create components for logoff , this should be just button in your header -> menu / nav